docs/en/sql-reference/window-functions/first_value.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /en/sql-reference/window-functions/first_value
+sidebar_label: first_value
+sidebar_position: 3
+---
+
+# first_value
+
+Returns the first non-NULL value evaluated within its ordered frame.
```

Comment:
`RESPECT NULLS` should be mentioned and the description and return type adjusted.